galaxy/iso2flux.xml

```diff
@@ -5,7 +5,7 @@
 <command><![CDATA[
 echo 'Working directory is '\$PWD;
 #if str( $tracing_model ) != None:
-#set $tracing_model_wext = str( $tracing_model ) + '.xlsx'
+#set $tracing_model_wext = str( $tracing_model ) 
 ln -s "$tracing_model" "$tracing_model_wext";
```

Comment:
This is the part where we could get rid of the linking I mentioned before, if iso2flux will accept files with a different extension.